src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -48,19 +53,40 @@ export class AssetsTable extends React.Component {
     });
   }
 
-  addDeleteButton() {
-    const newAssetsList = this.props.assetsList.map((asset) => {
+  getNextFocusElementOnDelete(assetToDelete) {
+    let rowToDelete = assetToDelete.row_key;
+    let focusAsset = assetToDelete;
+    let focusRowButton = this.trashcanRefs[assetToDelete.id];
+
+    if (this.props.assetsStatus.text === AssetsConstants.SUCCESS_FILE_DELETE) {
```

Comment:
Is this function (in theory) ever called if this isn't the case? Also, the comparison of text is a bit worrisome especially in the case that the text is translated. Is there some sort of unique state/status/prop/code/you-get-the-gist that we could use instead?

Reply:
Yes, it will still be called because regardless of the response status we need to know where to send focus when the status alert is closed out of. The text comparison is using the response body text, not text that will ever been shown to the user / internationalized. (The ones that now live in the constants/assets.js file). But I understand the concern, I'll test out if I can use response code or something else here.

Reply:
Oh, I see! I think that `assetsStatus` has a `text` field is what confused me. That's your status code, right? I don't want to bloat your PR, but I think that if `text` were renamed to `statusCode` or `type` (or something else), it would be clearer. To me, `text` indicates that it's displayed to the user.